elements/pfe-styles/src/pfe-typography-classes.scss

```diff
@@ -1,14 +1,15 @@
 @import "../../pfe-sass/pfe-sass";
 
 // TITLES
-@each $size in (6xl, 5xl, 4xl, 3xl, 2xl, xl, lg, md, sm, xs) {
+@each $size in ("6xl", "5xl", "4xl", "3xl", "2xl", "xl", "lg", "md", "sm", "xs") {
   .pfe-title--#{$size}  {
-      @extend %pfe-title--#{$size}; 
+      @include pfe-title($sizing: #{$size}, $important: true); 
```

Comment:
```suggestion
      @include pfe-title($size, $important: true); 
```

This feels a little easier to read to me ^

Reply:
I went with $sizing to avoid a conflict with the $size var in the for each loop in the typography_classes.scss

![image](https://user-images.githubusercontent.com/456863/106674619-992d4700-6581-11eb-9dc6-5eff7f4840b9.png)

Reply:
I don't think you understood my feedback.  I was referring to leaving off the input variable name.  You don't have to prefix all your mixin inputs with their variable names unless you're using them out of order.  I find some mixins easier to read in use when instead of `pfe-typography($sizing: $size)` you use `pfe-typography($size)`.  It's the same result, one is just simpler to mentally grok.